components/merkledb/tests/proof_map_index.rs

```diff
@@ -142,35 +154,42 @@ fn index_data(
     btree_map(array::uniform32(key_bytes), any::<u64>(), sizes)
 }
 
+fn data_for_absent(key_bytes: RangeInclusive<u8>) -> impl Strategy<Value = Vec<Key>> {
+    vec(array::uniform32(key_bytes), 20)
```

Comment:
Nit: `vec(array::uniform32(key_bytes).prop_map(Key), 20)` is shorter and more understandable.